src/components/TalentPage/TalentPageLayout.tsx

```diff
@@ -0,0 +1,238 @@
+import React, { FC, FormEvent, Suspense, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Controller, useForm } from 'react-hook-form';
+import { useMemo } from 'react';
+import {
+	Wrapper,
+	Title,
+	Input,
+	BorderStyled,
+	MainBlockWrapper,
+	Label,
+	Button,
+	ButtonBlock,
+	SkillsButtonsBlock,
+	WrapperSidePanel,
+	InputContainer,
+	IconSearch,
+	ProfileBlock,
+	PaginationBlock,
+} from './TalentPageLayout.style';
+import { ImSearch } from 'react-icons/im';
+import { ISkill } from 'components/jobPost/interfaces';
+import { skillsMock } from 'components/jobPost/dataChanges';
+import { Filter, ICategoryBE, SearchSubmitForm } from './interfaces';
+import Select from 'react-select';
+import { selection } from 'components/jobPost/dataChanges';
+import { BsArrowLeftCircle } from 'react-icons/bs';
+import { BsArrowRightCircle } from 'react-icons/bs';
+import MyHiresCompany from './myhires/MyHiresCompany';
+import MySavedTalent from './mysaved/MySavedTalent';
+import FilterProfileUser from './FilterProfileUser';
+import { useGetFilterProfileQuery } from 'service/httpService';
+import Pagination from './Pagination';
+
+const TalentPageLayout: FC = () => {
+	const { t } = useTranslation();
+	const [skillsOption, setSkillsOptions] = useState<ISkill[]>(skillsMock);
+	const [select, setSelect] = useState<ICategoryBE>();
+	const [search, setSearch] = useState<string>('');
+	const [showFilterList, setShowFilterList] = useState<boolean>(true);
+	const [active, setActive] = useState<{ [name: string]: string }>();
+	const [currentPage, setCurrentPage] = useState<number>(1);
+
+	const filteredSkills = skillsOption.filter(s => s.value);
```

Comment:
you make filter with each rerender, its bad approach

Reply:
On model from UI design we don't have button that filter all element and there is no different way.

Reply:
its not about ui, we cant filter in each rerender, move to hooks, backend , so on

Reply:
fixed

Reply:
but what was changed?

Reply:
changes in last commit https://github.com/ZenBit-Tech/Devs_Heads_fe/pull/41/commits/fb5f120a43ec8497885a07f48f30440888e63de5